src/control-plane/backend/lambda/api/common/utils.ts

```diff
@@ -367,6 +367,44 @@ function getValueFromStackOutputSuffix(pipeline: IPipeline, stackType: PipelineS
   return `#.${stackName}.${suffix}`;
 }
 
+function getInfoFromPipelineStatus(status: PipelineStatus, stackType: PipelineStackType, key: string): string {
```

Comment:
rename it to `getStackOutputFromPipelineStatus` ?

Reply:
done